### ISSUE TITLE
Return raw data

### ISSUE THREAD

**Issue by saleh199** [guyonroche/exceljs#711]
I have a normal data like this in my excel sheet "1-3" but when I read it by **exceljs** library it comes as a date which is wrong.

Imagen that we have these data in an excel sheet

    |  A  |  B  |  C  |
    -------------------
    | 1-3 | 3-4 | 1-2 |

And the code like this `worksheet.getCell('A1').value` or `worksheet.getCell('A1').text` and the return data comes `Wed Jan 03 2018 03:00:00 GMT+0300 (+03)`

**Comment by hayzem** [comment]
I am facing a similiar issue too. Is there a way to disable date detection/conversion?

**Comment by angelmartinez-wz** [comment]
I have tested with the following cases:

```
ba 12345678
12345678 ba
ba 12345678 ba
Ba-12345678
Ba 12097648
Ba 123456789
ba12345678ba
```

But only the second one is processed correctly, @saleh199 @hayzem does anyone could handle this?

**Comment by hayzem** [comment]
@angelmartinez-wz in our case, we ended up using different notation where dash is not in the cell value.

**Comment by kpittman-securus** [comment]
That's a decent workaround when you control the data, but if you don't control the data then you're kind of stuck here. "2020-00001" is not a date, and I want to tell exceljs to stop trying to coerce it into one. What if we could just pass a `raw: true` parameter in an options object somewhere?

I have a CSV and the first entry is an ID, like `2020-00001`. My parse code is simple: `new Workbook().csv.readFile(filename)`. When I read the file in and try to process the data, I get an error because `2020-00001` has somehow been coerced to `Thu Aug 20 1903 00:00:00 GMT-0600 (Central Daylight Time)`.

I tried setting `{dateFormats: ["MM/dd/yyyy"]}` to see if it changes anything (maybe `2020-` is tricking exceljs into thinking it's a year?). But, this just causes MORE of my fields in the CSV to be parsed as dates. `123` becomes `Tue Dec 01 2020 00:00:00 GMT-0600 (Central Standard Time)`.

I found the link from the exceljs documentation pointing to dayjs... https://github.com/iamkun/dayjs/blob/HEAD/docs/en/Plugin.md#customparseformat

I see that `Do` maps to ordinal date... like `1st`,  `2nd`... etc. I know that string will never occur in my source data, so I set `{dateFormats: ["Do"]}`. The parser now works as intended, and doesn't coerce non-dates into dates.

TL;DR: my short-term workaround is to set `{dateFormats:["..."]}` in your parse options and use a `dayjs` format that you know will not occur in your source data. Only works if you can know your source data ahead of time.

Since I'm working only with CSV data, I'll be switching to the `csv` package as a longterm fix.
